types/spotify-api/index.d.ts

```diff
@@ -954,7 +954,8 @@ declare namespace SpotifyApi {
         /**
          * The type of the album: one of “album”, “single”, or “compilation”.
          */
-        album_type: 'album' | 'single' | 'compilation';
+        album_type: 'album' | 'single' | 'compilation' |
+                    'ALBUM' | 'SINGLE' | 'COMPILATION';
```

Comment:
```suggestion
        album_type: 'album' | 'single' | 'compilation';
```
This should still be reverted, as only the `RecommendationsFromSeeds` returns these types.